TACC-Cloud/geoapi
# task/WG-256: use tapisv3 file operations

## Overview: ##

This PR updates
* to tapis v3 for file operations  [WG-256](https://tacc-main.atlassian.net/browse/WG-256)
* updates how we get project's users from DS's /api/projects endpoint   [WG-257](https://tacc-main.atlassian.net/browse/WG-257)
* adds a fixture to use as we wait for the deployed tapisv3-based DS [WG-257](https://tacc-main.atlassian.net/browse/WG-257)

This PR also:
 * adds a `restart-workers` and `help` command to Makefile
 * adds some refactoring to getting users and related todos.

## Related Jira tickets: ##

* [WG-256](https://tacc-main.atlassian.net/browse/WG-256)
* [WG-257](https://tacc-main.atlassian.net/browse/WG-257)

## Testing Steps: ##
1. Test with angular hazmapper from `feature/tapisv3` branch.
2. Create a map and then manually add files
4. Create a second map and select a folder to automatically import files from the DS project `TEST: Hazmapper/TapisV3 project`
5. Check that the map project has the list of users for `TEST: Hazmapper/TapisV3 project`
6. run 'make help' and 'make restart-workers'
